src/StackExchange.Redis/PhysicalBridge.cs

```diff
@@ -20,7 +20,7 @@ internal sealed class PhysicalBridge : IDisposable
 
         private const int ProfileLogSamples = 10;
 
-        private const double ProfileLogSeconds = (ConnectionMultiplexer.MillisecondsPerHeartbeat * ProfileLogSamples) / 1000.0;
+        private const double ProfileLogSeconds = (1000 /* ms */ * ProfileLogSamples) / 1000.0;
```

Comment:
Why multiply by a thousand just to divide again?

Reply:
This was really just showing intent of the net change without doing a bigger rename, etc. - can be simplified for sure.